cloudformation_templates/aws_jenkins.json

```diff
@@ -122,13 +126,24 @@
       }
     },
 
-    "Route53RecordSet": {
+    "CIRoute53RecordSet": {
+      "Type": "AWS::Route53::RecordSet",
+      "Properties": {
+        "HostedZoneName": {"Ref": "HostedZoneName"},
+        "Name": {"Fn::Join": [".", [{"Ref": "Subdomain"}, {"Ref": "HostedZoneName"}]]},
+        "Type": "A",
+        "ResourceRecords": [
+          {"Ref": "ElasticIP"}
+        ],
+        "TTL": "300"
+      }
+    },
+
+    "LogsRoute53RecordSet": {
       "Type": "AWS::Route53::RecordSet",
       "Properties": {
         "HostedZoneName": {"Ref": "HostedZoneName"},
-        "Name": {"Fn::Join": ["",[
-          {"Ref": "Subdomain"}, ".", {"Ref": "RootDomain"}, "."
-         ]]},
+        "Name": {"Fn::Join": [".", ["*", {"Ref": "LogsSubdomain"}, {"Ref": "HostedZoneName"}]]},
```

Comment:
Right now it is, and it's set in the stack variables, so we'd need another commit if we decide to change it to something else.